README.md

```diff
@@ -1,35 +1,78 @@
-# Gluco Check
+# Welcome to Gluco Check
 
-formerly Nightscout Assistant / Nightscout Status
+#### formerly [Nightscout Status](https://github.com/nielsmaerten/nightscout-assistant) 
+[![CodeFactor Grade](https://img.shields.io/codefactor/grade/github/nielsmaerten/gluco-check/main?style=flat-square)](https://www.codefactor.io/repository/github/nielsmaerten/gluco-check/)
+[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/nielsmaerten/gluco-check/Build,%20Test,%20Lint,%20Deploy?style=flat-square)](https://github.com/nielsmaerten/gluco-check/actions?query=workflow%3A%22Build%2C+Test%2C+Lint%2C+Deploy%22)
+[![Codecov](https://img.shields.io/codecov/c/github/nielsmaerten/gluco-check?style=flat-square)](https://codecov.io/gh/nielsmaerten/gluco-check)
 
-## Hey Google, talk to Gluco Check
+## Getting started
 
-> "Blood sugar is 102 and rising slowly as of 4 minutes ago."
+<!-- TODO: Update URL to Production version -->
+1. [Link your Nightscout site](https://nightly.diabase.app)
+2. [Ask for beta access](https://gitter.im/nielsmaerten)
+3. Say: **"Hey Google, talk to Gluco Check"**  
 
-## Bèta testing is now open
+## Things you can ask Gluco Check
+
+### Blood sugar
+
+Hey Google, ask Gluco Check my blood sugar
```

Comment:
possible alts:
Hey Google, ask Gluco Check my BG
Hey google, ask gluco check what's my [metric] (the what's is for sure grammatically incorrect but I can see myself attempting it)

Reply:
Would this be correct?
- Hey Google, ask Gluco Check to read my [metric] 
- Hey Google, ask Gluco Check to tell me my [metric]
That would also follow Google's recommended verb-subject pattern